### PR TITLE
Add missing S3method() declarations

### PR DESCRIPTION
**Pull request by HenrikBengtsson**

### CODE REVIEW

**Comment by HenrikBengtsson** [comment]
For commit ae9e4f4, I've added automatic counters to titles and filenames to make it easy to keep track of number of posts done and match the counts toward the title/filename.

I also robustified the test script such that it make sure they actually can be done, e.g. `hasEmail`.

I can post files, but I cannot download them:
```xml
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>A0...91</RequestId>
<HostId>
Wg1V...MWrc
</HostId>
</Error>
```

That's all the time I have for this now.

PS. I'm testing this on Windows.

File: NAMESPACE [hunk 1]
```diff
@@ -3,3 +3,7 @@ import(RJSONIO)
 importFrom(utils, packageDescription)
 
 exportPattern("^[[:alpha:]]+")
+
+S3method("pbGetDevices", "default")
```
Comment (eddelbuettel) [inline NAMESPACE]: AFAICT that is not needed as the wildcard `exportPattern` takes care of this.  If I do `ls(pos=p)` for the given `p`, I see the symbols already.

Comment (HenrikBengtsson) [inline NAMESPACE]: S3method() != export().  It's to declare that a function with name `<name>.<class>` is an S3 method `<name>` for class `<class>`.  This helps S3 and S4 method dispatching to work properly.  This is the only way R can know that, e.g., `t.test()` is not method `t` for class `test`.

Comment (HenrikBengtsson) [inline NAMESPACE]: Also, S3method() becomes important when a package is not attached, but only loaded, which happens when other packages import such functions.  Typical symptoms are hard to track errors.

Comment (eddelbuettel) [inline NAMESPACE]: Taa.

**Pull request merged**